pkg/bastion/acl.go

```diff
@@ -36,7 +40,7 @@ func checkACLs(user dbmodels.User, host dbmodels.Host) string {
 
 	// deny by default if no shared ACL
```

Comment:
can you update this comment to reflect what is now happening?

Reply:
fixed comment